src/integrations/admin/addon-installation/dialog-integration.php

```diff
@@ -94,6 +109,22 @@ public function throw_no_owned_addons_warning() {
 			'</p></div>';
 	}
 
+	/**
+	 * Displays success message.
+	 *
+	 * @returns void
+	 */
+	public function generate_success_box() {
+		\wp_localize_script(
```

Comment:
If you're not putting any data in the localization object, then it does not need to be output at all.

I think it would be better to generate the full admin URL here and pass it to the component to be used though.